#Expyriment 0.9.0 (Python 3.6.8), .xpe-file, coding: cp1252
#date: Sun Apr 28 2019 13:53:48
#sha1: None
#modules: 
#display: size=(1920, 1080), window_mode=False, open_gl=2
#os: uname_result(system='Windows', node='DESKTOP-OVOASGR', release='10', version='10.0.17134', machine='AMD64', processor='Intel64 Family 6 Model 94 Stepping 3, GenuineIntel')
Time,Type,Event,Value,Detail,Detail2
11333,design,log,
#design: #exp: Tutorial
#design: #dvn: projection,date,fictional,event_to_projection_distance,before_is_left,pressed_key,good_answer,RT
#design: block_cnt,block_id,trial_cnt,trial_id,Date,Fictional
#design: 0,0,0,0,1985,False
#design: 0,0,1,1,1988,True
#design: 0,0,2,2,1991,False
#design: 0,0,3,3,1994,True
#design: 0,0,4,4,1997,False
#design: 0,0,5,5,2000,True
#design: 0,0,6,6,2003,False
#design: 0,0,7,7,2006,False
#design: 0,0,8,8,2009,False
#design: 0,0,9,9,2012,False
#design: 0,0,10,10,2015,True
#design: 0,0,11,11,2018,False
#design: 0,0,12,12,2019,True
#design: 0,0,13,13,2019,False
#design: 0,0,14,14,2020,False
#design: 0,0,15,15,2023,True
#design: 0,0,16,16,2026,False
#design: 0,0,17,17,2029,True
#design: 0,0,18,18,2032,False
#design: 0,0,19,19,2035,False
#design: 0,0,20,20,2038,True
#design: 0,0,21,21,2041,False
#design: 0,0,22,22,2044,True
#design: 0,0,23,23,2047,False
#design: 0,0,24,24,2050,True
#design: 0,0,25,25,2053,False
11333,design,logged,
11333,Experiment,started
11334,Stimulus,presented,26
12338,Stimulus,presented,29
15947,Keyboard,received,13,wait
15947,Stimulus,presented,26
16951,Stimulus,presented,32
19486,Keyboard,received,13,wait
19486,Stimulus,presented,26
20490,stderr,received,'Traceback (most recent call last):'
20491,stderr,received,'  File "scripts\\tutorial.py", line 364, in <module>'
20492,stderr,received,'    for trial in blocks.trials[0,1,2,3] :'
20494,stderr,received,"NameError: name 'blocks' is not defined"
